xarray/core/computation.py

```diff
@@ -1013,6 +1013,7 @@ def earth_mover_distance(first_samples,
     if dask == "parallelized":
         if dask_gufunc_kwargs is None:
             dask_gufunc_kwargs = {}
+        dask_gufunc_kwargs = dask_gufunc_kwargs.copy()
```

Comment:
```suggestion
        else:
            dask_gufunc_kwargs = dask_gufunc_kwargs.copy()
```

Reply:
Thanks for the suggestion @dcherian ! I have commited the change.